tests/test_parser.py

```diff
@@ -94,6 +94,26 @@ def test_propagate_positions(self):
         r = g.parse('a')
         self.assertEqual( r.children[0].meta.line, 1 )
 
+    def test_propagate_positions2(self):
+        g = Lark("""start: a
+                    a: b
+                    ?b: "(" t ")"
+                    !t: "t"
+                 """, propagate_positions=True)
+
+        start = g.parse("(t)")
+        a ,= start.children
+        t ,= a.children
+        assert t.children[0] == "t"
+
+        assert t.meta.column == 2
+        assert t.meta.end_column == 3
+
+        assert start.column == a.column == 1
```

Comment:
These are deprecated. Also, I would suggest using `self.assertEqual` for better output on failure.

Reply:
They're not meant to fail ;)

Reply:
Then why even test :-P

Reply:
So that when we refactor, or add new features, we know that we didn't break anything. I'm not against using assertEquals, I'm just don't see why it matters. Debugging would usually take a lot longer than replacing the offending assert, or setting a breakpoint on it.

Reply:
I mean yeah it isn't that important. More important (and why I am not approving with this review) is the usage of `Tree.column`, which is [deprecated](https://github.com/lark-parser/lark/blob/eb2b0b91663ff29a3c2a15e988178b53129d2436/lark/tree.py#L150).

Reply:
Oh okay, I got it now. Funny I didn't get any warnings.

Reply:
Anyway, good catch. Fixed now.